content/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "Munro Mortgages"
+description: "Here to help"
+
+theme_version: '2.8.2'
+cascade:
+  featured_image: '/images/hero.jpg'
+---
+
+Welcome to my blog with some of my work in progress. This site is here for you to get to know me better, what I care about, and the sort of work I can help you with.
```

Comment:
I copied this from their Example Site

NOTE: the `featured_image` is pointing at the file I added in `static/images/hero.jpg`